src/jsTest/kotlin/react/dom/test/Simulate.kt

```diff
@@ -0,0 +1,15 @@
+package react.dom.test
```

Comment:
this package (react.dom.test) has been deleted in the main branch. We don't need in our source as it's a dependency.